cmd/root.go

```diff
@@ -52,9 +52,12 @@ var rootCmd = &cobra.Command{
 		skipDeps, _ := cmd.Flags().GetBool("skip-deps")
 		diffRun, _ := cmd.Flags().GetBool("diff-run")
 
+		askUserToConfirm, _ := cmd.Flags().GetBool("asking-for-confirmation")
```

Comment:
Think having an explicit default would be best. Also a shorthand would be good for such a log flag name. See here https://github.com/DataReply/korgi/blob/master/cmd/root.go#L89

Reply:
Added in e4b261f681b7512c167d85049e8f64f701f9e688